examples/main.rs

```diff
@@ -26,27 +26,27 @@ extern crate webkit2gtk;
 
 #[cfg(feature = "v2_4")]
 use glib::ToVariant;
-use gtk::{ContainerExt, Inhibit, WidgetExt, Window, WindowType};
-use webkit2gtk::{SettingsExt, WebContext, WebContextExt, WebView, WebViewExt, WebViewExtManual};
+use gtk::{Inhibit, Window, WindowType, prelude::{ContainerExt, WidgetExt}};
```

Comment:
```suggestion
use gtk::{Inhibit, Window, WindowType, prelude::*};
```
use prelude directly